cli/integration_tests/single_package/run.t

```diff
@@ -21,10 +21,10 @@ Run a second time, verify caching works because there is a config
   \xe2\x80\xa2 Running build (esc)
    INFO  \xe2\x80\xa2 Remote caching disabled (esc)
   build: cache hit, replaying output d415b27c3e54b6ed
-  build: 
+  build:  
```

Comment:
This extra space looks like a behavior change. Is this intentional?

Reply:
intentional, unfortunately. could fix it with a larger refactor, but it doesn't seem worth it. 

comments here: https://github.com/vercel/turborepo/pull/2126/files#diff-abd1634cb17583d01043d889209fd82162346e24bfb9ee2935fbd2c8a2b3e826R351